From 72b3f7dc0542ee33966a0cf3fc605672a9bb667a Mon Sep 17 00:00:00 2001
From: Pierre-Louis Bossart <pierre-louis.bossart@linux.intel.com>
Date: Mon, 16 Apr 2018 13:20:32 -0500
Subject: [PATCH 1212/1240] ASoC: SOF: enable DEBUG by default

better to use dynamic debug for products but it's really convenient

Signed-off-by: Pierre-Louis Bossart <pierre-louis.bossart@linux.intel.com>
---
 sound/soc/sof/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/sound/soc/sof/Makefile b/sound/soc/sof/Makefile
index ece9300..b759554 100644
--- a/sound/soc/sof/Makefile
+++ b/sound/soc/sof/Makefile
@@ -1,5 +1,7 @@
 # SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause)
 
+ccflags-y += -DDEBUG
+
 snd-sof-objs := core.o ops.o loader.o ipc.o pcm.o pm.o debug.o topology.o\
 		control.o trace.o compressed.o
 snd-sof-spi-objs := hw-spi.o
-- 
2.7.4

